docs/scripts/buildApi.js

```diff
@@ -199,7 +204,11 @@ function run() {
   const components = findComponents(path.resolve(rootDirectory, args[2]));
 
   components.forEach(component => {
-    buildDocs({ component, pagesMarkdown });
+    buildDocs({ component, pagesMarkdown }).catch(error => {
```

Comment:
I'm surprised it hasn't exploded yet 🚀🌕. This is like 200 concurrent promises 💪.

Reply:
I will celebrate the day uncaught promises will just throw. The current state of async errors in node is really annoying.